Patches/Rimeffect Core/Mechpawn_ME.xml

```diff
@@ -0,0 +1,247 @@
+				</value>
+			</li>
+    			<li Class="PatchOperationReplace">
+     				 <xpath>/Defs/ThingDef[defName="RE_Mechanoids_YMIR"]/race/baseBodySize</xpath>
+      				<value>
+        				<baseBodySize>2</baseBodySize>
+      				</value>
+    			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[defName="RE_Mechanoids_FENRIS"]/statBases</xpath>
+				<value>
+					<AimingAccuracy>1.0</AimingAccuracy>
+					<ShootingAccuracyPawn>1.0</ShootingAccuracyPawn>
+					<MeleeDodgeChance>1</MeleeDodgeChance>
+					<MeleeCritChance>0.28</MeleeCritChance>
+					<MeleeParryChance>0.15</MeleeParryChance>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[defName="RE_Mechanoids_LOKI"]/statBases</xpath>
+				<value>
+					<AimingAccuracy>1.0</AimingAccuracy>
+					<ShootingAccuracyPawn>1.0</ShootingAccuracyPawn>
+					<MeleeDodgeChance>0.5</MeleeDodgeChance>
+					<MeleeCritChance>0.21</MeleeCritChance>
+					<MeleeParryChance>0.2</MeleeParryChance>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[defName="RE_Mechanoids_YMIR"]/statBases</xpath>
+				<value>
+					<AimingAccuracy>1.1</AimingAccuracy>
+					<ShootingAccuracyPawn>1.1</ShootingAccuracyPawn>
+					<MeleeDodgeChance>0.1</MeleeDodgeChance>
+					<MeleeCritChance>0.8</MeleeCritChance>
+					<MeleeParryChance>0.5</MeleeParryChance>
+					<CarryWeight>400</CarryWeight>
+			                	<CarryBulk>1000</CarryBulk>
+					<MaxHitPoints>500</MaxHitPoints>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RE_Mechanoids_LOKI"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>14</ArmorRating_Blunt>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RE_Mechanoids_LOKI"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>9</ArmorRating_Sharp>
+				</value>
+			</li>		
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RE_Mechanoids_FENRIS"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>16</ArmorRating_Blunt>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RE_Mechanoids_FENRIS"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>6</ArmorRating_Sharp>
+				</value>
+			</li>		
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RE_Mechanoids_YMIR"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>75</ArmorRating_Blunt>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RE_Mechanoids_YMIR"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>22</ArmorRating_Sharp>
+				</value>
+			</li>	
+
+    <li Class="PatchOperationReplace">
+      <xpath>Defs/ThingDef[defName="RE_Mechanoids_FENRIS"]/tools</xpath>

```

Comment:
The Fenris mechs have a sort of EMP/shock discharger melee attack it may be interesting to include.

Reply:
Is it in the original Mass effect mod? If it isn't, I'll still take a look, probably only in the form a melee attack tho.

Reply:
It's done, for some godforsaken reason the preview here isn't showing the updated file, rest assured that I've updated all the problem files in my fork.

Reply:
I'm unable to use taser as a melee attack, do you know what's the criteria for a damage to be useable as a melee capacity?

Reply:
> The Fenris mechs have a sort of EMP/shock discharger melee attack it may be interesting to include.

Updated Fenris taser attack, let me know if it works.